test/src/main/java/org/apache/accumulo/test/functional/CloneTestIT_SimpleSuite.java

```diff
@@ -367,8 +367,9 @@ public void testCloneSystemTables() {
       for (int i = 0; i < sysTables.length; i++) {
         var sysTable = sysTables[i];
         var cloneTableName = tableNames[i];
-        assertThrows(Exception.class, () -> client.tableOperations().clone(sysTable.tableName(),
-            cloneTableName, CloneConfiguration.empty()), () -> "Table:" + sysTable.tableName());
+        assertThrows(AccumuloException.class, () -> client.tableOperations()
+            .clone(sysTable.tableName(), cloneTableName, CloneConfiguration.empty()),
+            () -> "Table:" + sysTable.tableName());
```

Comment:
Was previously `Exception` because different exceptions could be thrown for diff system tables. #5711 fixed this, so updated the test here